### PR TITLE
Spanish Translation Update

### PR DESCRIPTION
**Pull request by jamperezmondragon**
I fixed typos, inconsistencies with verb conjugations, inconsistencies, random stuff that didn't make sense. 

Screenshot from 2024-07-14 02-34-17

 I am a native Spanish speaker, and I have experience doing English-Spanish translations.

### CODE REVIEW

File: selfdrive/ui/translations/main_es.ts [hunk 1]
```diff
@@ -1024,43 +1024,43 @@ Esto puede tardar hasta un minuto.</translation>
...
+        <translation>Se recomienda el modo estándar. En el modo agresivo, openpilot seguirá más cerca a los autos delante suyo y será más agresivo con el acelerador y el freno. En modo relajado, openpilot se mantendrá más alejado de los autos delante suyo. En automóviles compatibles, puede recorrer estas personalidades con el botón de distancia del volante.</translation>
     </message>
     <message>
         <source>openpilot defaults to driving in &lt;b&gt;chill mode&lt;/b&gt;. Experimental mode enables &lt;b&gt;alpha-level features&lt;/b&gt; that aren&apos;t ready for chill mode. Experimental features are listed below:</source>
-        <translation>openpilot por defecto conduce en &lt;b&gt;modo chill&lt;/b&gt;. El modo Experimental activa &lt;b&gt;recursos de nível-alfa&lt;/b&gt; que no están listos para el modo chill. Los recursos del modo expeimental están listados abajo:</translation>
+        <translation>openpilot por defecto conduce en &lt;b&gt;modo chill&lt;/b&gt;. El modo Experimental activa funcionalidades en fase experimental, que no están listas para el modo chill. Las funcionalidades del modo expeimental están listados abajo:</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_es.ts:1031]: does this remove the bolding/italics?

Comment (jamperezmondragon) [reply selfdrive/ui/translations/main_es.ts:1031]: Ops you're right, I'll change that.

File: selfdrive/ui/translations/main_es.ts [hunk 2]
```diff
@@ -721,15 +721,15 @@ Esto puede tardar hasta un minuto.</translation>
     <name>Sidebar</name>
     <message>
         <source>CONNECT</source>
-        <translation>CONNECT</translation>
+        <translation>CONECTAR</translation>
```
Comment (MarcoTheDingo) [inline selfdrive/ui/translations/main_es.ts:724]: I believe we would want it to stay as CONNECT since that's the proper name of comma's service.

**Review by MarcoTheDingo — COMMENTED** [review]
Looks good, thanks for updating it! Just a small feedback about Connect being a service and probably should remain the same since it's a proper name.

File: selfdrive/ui/translations/main_es.ts [hunk 3]
```diff
@@ -804,7 +804,7 @@ Esto puede tardar hasta un minuto.</translation>
     <name>SoftwarePanel</name>
     <message>
         <source>Updates are only downloaded while the car is off.</source>
-        <translation>Apague el auto para descargar actualizaciones.</translation>
+        <translation>Las actualizaciones solo se descargan cuando con el auto apagado.</translation>
```
Comment (MarcoTheDingo) [inline selfdrive/ui/translations/main_es.ts:807]: I had to shorten this line a lot to fit in the UI. Can you make this any shorter?

Comment (jamperezmondragon) [reply selfdrive/ui/translations/main_es.ts:807]: Sure, I'll shorten it (also thank you I just realized I forgot to erase something here when translating ops). 

What about "Actualizaciones solo se descargan con el auto apagado." I think this fits in the UI.

File: selfdrive/ui/translations/main_es.ts (hunk 2 above)
Comment (jamperezmondragon) [reply selfdrive/ui/translations/main_es.ts:724]: That makes sense, I actually had my doubts on weather to change that or not, I basically did the same as the french translation, but maybe that one is mistaken. I'll update that in my other PR.

**Pull request merged**